lib/containers/common.pm

```diff
@@ -34,6 +34,14 @@ sub install_podman_when_needed {
         if ($host_os eq 'centos') {
             assert_script_run "dnf -y install @pkgs", timeout => 160;
         }
+        elsif ($host_os eq 'ubuntu') {
+            my $version_id  = script_output("grep VERSION_ID /etc/os-release | cut -c13- | tr -d '\"'");
```

Comment:
This is exactly the case we're trying to avoid with the `get_os_release()`. I suggest to call it from here instead of passing it as argument to the routine.

Reply:
Thing is that the `get_os_release` does not return exactly what i need here. for Ubuntu returns an extented version description like `20.04.1 LTS (Focal Fossa)`. What i need is only the output of VERSION_ID which gives me the short version. I thought that i do not need to put it in the `get_os_release` for this purpose or i do not need to call the sub and then fetch only the part that i want to use. i think it is ok as it is. I will resolve this and feel free to reopen it if you think otherwise.